arch/toolchains/ecmwf-hpc2020-nvhpc.cmake

```diff
@@ -37,14 +37,6 @@ set( OpenACC_Fortran_FLAGS "-acc=gpu -mp=gpu -gpu=cc80,lineinfo,fastmath" CACHE
 # Enable this to get more detailed compiler output
 # set( OpenACC_Fortran_FLAGS "${OpenACC_Fortran_FLAGS} -Minfo" )
 
-####################################################################
```

Comment:
Again, not sure the NEC changes require us to change Nvidia configurations as such?